Code/Mantid/Framework/DataHandling/test/CheckMantidVersionTest.h

```diff
@@ -0,0 +1,138 @@
+#ifndef MANTID_DATAHANDLING_CHECKMANTIDVERSIONTEST_H_
+#define MANTID_DATAHANDLING_CHECKMANTIDVERSIONTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidDataHandling/CheckMantidVersion.h"
+
+using Mantid::DataHandling::CheckMantidVersion;
+using namespace Mantid::API;
+
+namespace {
+/**
+ * Mock out the version calls of this algorithm
+ */
+class MockedCheckMantidVersion : public CheckMantidVersion {
+public:
+  MockedCheckMantidVersion(std::string currentVersion, std::string gitHubVersion) : CheckMantidVersion(), CurrentVersion(currentVersion),
+    GitHubVersion(gitHubVersion){}
+
+  std::string CurrentVersion;
+  std::string GitHubVersion;
+private:
+  virtual std::string getVersionsFromGitHub(const std::string &url) {
+    //the initial assignment of the value to url is just to suppress a compiler warning
+    std::string outputString(url);
+    outputString ="{\n"
```

Comment:
The warning was about the fact I did not use the url variable, rather than the initial initialisation of outputstring.

I could probably delete url, but as it was a linux only error and I was on windows I took a safer approach.